diff --git a/wrappers/python/CMakeLists.txt b/wrappers/python/CMakeLists.txt
index ad4ccdc..910ddd1 100644
--- a/wrappers/python/CMakeLists.txt
+++ b/wrappers/python/CMakeLists.txt
@@ -32,8 +32,12 @@ target_link_libraries(cython_freenect freenect_sync ${PYTHON_LIBRARIES})
 include_directories(${PYTHON_INCLUDE_PATH} ../c_sync/ ${NUMPY_INCLUDE_PATH})
 
 # Install the extension
+execute_process( COMMAND python -c "from distutils.sysconfig import get_python_lib; print get_python_lib()" OUTPUT_VARIABLE PYTHON_SITE_PACKAGES OUTPUT_STRIP_TRAILING_WHITESPACE)
 install(TARGETS cython_freenect
-  DESTINATION ${PYTHON_SITE_PACKAGES})
+  DESTINATION ${PYTHON_SITE_PACKAGES}
+  LIBRARY DESTINATION ${PYTHON_SITE_PACKAGES})
+
+
 
 # TODO: decide on what to do with demo_ scripts and were to install
 #       them
